test/Microsoft.Dnx.DesignTimeHost.FunctionalTests/Infrastructure/DthFunctionalTestFixture.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using System.Linq;
+using Microsoft.Dnx.CommonTestUtils;
+
+namespace Microsoft.Dnx.DesignTimeHost.FunctionalTests.Infrastructure
+{
+    public class DthFunctionalTestFixture : DnxRuntimeFixture
+    {
+        private readonly DisposableDir _context;
+        private readonly int[] _ports;
+
+        private int _nextPortIdx;
+
+        public DthFunctionalTestFixture() : base()
+        {
+            _ports = Enumerable.Range(1774, 20).ToArray();
```

Comment:
It's a good suggestion, but I'd like to avoid the cover complicated test code here. Should this port conflict happen it is easy to detected and corrected.